squad/api/urls.py

```diff
@@ -7,7 +7,7 @@
 from . import ci
 
 
-slug_pattern = '[a-z0-9_.-]+'
+slug_pattern = '[A-Za-z0-9_.-]+'
```

Comment:
aaaah ... that's why. I think the right fix for this is to have a single definition of what's a valid slug. IMO we should keep it in `squad.core.models`, and reuse it here